executor/executor.go

```diff
@@ -1607,7 +1655,8 @@ func (e *SortExec) Next() (*Row, error) {
 	return row, nil
 }
 
-// TopnExec implements a top n algo.
+// TopnExec implements a top n algorithm, it is built from a select statement with ORDER BY and LIMIT.
+// Instead of sort all the rows fetched from table, it only keeps the top-n element in a heap to reduce memory usage.
```

Comment:
->Instead of sorting all the rows fetched from the table, it keeps the Top-N element only in a heap to reduce memory usage.